data/json/items/ammo/300BLK.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "id": "300BLK_supersonic",
```

Comment:
These IDs might start getting messy if new 300BLK types are introduced. Consider `300blk_TYPE` for standard loads and `300blk_ss_TYPE` for subsonic loads where TYPE is the projectile type eg `300blk_fmj` and `300blk_ss_jhp`.